### DISCUSSION TITLE
initial state for ARDESPOT

### DISCUSSION THREAD

**Discussion by Etwari3** [JuliaPOMDP/POMDPs.jl#472] [Debugging Help] [answered]
Hello,

I formulated the initial state for ARDESPOT as

    initialstate(m::PipeCS) = Normal(0.5, 0.3);

However, I got this error 

    MethodError: no method matching initialstate(::PipeCS)
      Closest candidates are:
        initialstate(::Union{MDP, POMDP}) 
        initialstate(::Any, ::Any) at deprecated.jl:70
        initialstate(::Union{OpaqueRLEnvMDP, OpaqueRLEnvPOMDP})

Is the formulation different from the BasicPOMCP?

Thank you.

**Comment by zsunberg** [comment, marked as answer]
Does it work if you replace it with
```julia
POMDPs.initialstate(m::PipeCS) = Normal(0.5, 0.3);
```
?

**Reply by Etwari3** [reply]
Yes, it worked. I have to define all components in a similar manner. Thank you, zsunberg.
